src/components/Settings/Settings.scss

```
@@ -73,6 +73,39 @@
     color: vars.$text-secondary;
     line-height: 1.5;
     margin-top: 6px;
+
+    // A recovery action offered inside the sentence that explains why it is
+    // needed ("could not load your voice — [Retry]"), so it flows with the
+    // text and inherits its size rather than sitting on its own row.
+    //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)